addons/website/static/tests/tours/powerbox_snippet.js

```diff
@@ -0,0 +1,76 @@
+/** @odoo-module **/
+
+import wTourUtils from "@website/js/tours/tour_utils";
+
+wTourUtils.registerWebsitePreviewTour("website_powerbox_snippet", {
```

Comment:
Can't we add Qunit/Hoot tests instead of Tour?

Reply:
Hello @msh-odoo , 

The changes we have made & testing are the part of the website frontend. as fas as I have research, there are no qunit/hoot are written for the frontend/website. to write qunit test for frontend we will require to generate mockUI as well, I didn't found any related utils. So i don't think right now it's good PR to add qunit/hoot, let's see if in near future we'll have the qunit for the frontend. ( No doubt, we can write the qunit for the backend view to test the same functinality, but this powerbox command present in website only so...)

Thanks : )

Reply:
Well, Tests are already available for the core part of the editor, just for information check ```web_editor/static/src/js/editor/odoo-editor/test/spec/powerbox.test.js``` but that is for testing core part of powerbox, so let's keep it as it is.